src/components/layout.css

```diff
@@ -1,5 +1,5 @@
 html {
-  font-family: sans-serif;
+  font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
@lannonbr How do we set an overall font-family in a styled component site? I'm also not really sure why I had to change it 3 times in this file 🤷‍♂️ ... thoughts?

Reply:
It looks like Styled components has a [createGlobalStyle](https://www.styled-components.com/docs/api#createglobalstyle) helper which you can have things not scoped so in the future we clean up the global CSS and then use that.

Reply:
Just made an issue to follow up on that later